rust/private/utils.bzl

```diff
@@ -238,6 +238,17 @@ def _expand_location(ctx, env, data):
         if directive in env:
             # build script runner will expand pwd to execroot for us
             env = env.replace(directive, "$${pwd}/" + directive)
+    for directive in ("$(execpaths ", "$(locations "):
```

Comment:
What about the singular variant of these?

Reply:
I left the existing handling of those alone, a few lines above this.